pkg/util/patroni/patroni.go

```diff
@@ -14,13 +14,15 @@ import (
 
 const (
 	failoverPath = "/failover"
+	configPath   = "/config"
 	apiPort      = 8008
 	timeout      = 30 * time.Second
 )
 
 // Interface describe patroni methods
 type Interface interface {
 	Failover(master *v1.Pod, candidate string) error
+	SetPostgresParameters(server *v1.Pod, options map[string]string) error
```

Comment:
Is this method dedicated only to params that require Postgres restart ?

Reply:
Not necessary.